src/components/PasswordDialog.vue

```diff
@@ -59,6 +59,13 @@ export default defineComponent({
 		NcPasswordField,
 	},
 
+	props: {
+		callback: {
```

Comment:
Requires a default like

```suggestion
		callback: {
			default: () => {},
```

Reply:
If it has a default, then it won't be undefined later on and the old behavior will never be called. I think that I tried to set `undefined` as default, but it failed.